File: python/twoSum.py
class Sol:
    def twoSum(self,nums,target):
        subArr = list(map(lambda v:target-v,nums))
        answer=[]
        for i in range(len(subArr)):
            try:
                index=nums.index(subArr[i])
                if index != i:
                    answer.append(i)
                    answer.append(index)
                    break
            except:
                pass
        return answer


s=Sol()
print(s.twoSum([1,2,3,4,5],3))

File: leetcode/twoSum.js
function sol(nums, target) {
  let answer = [];
  let subArr, index;
  subArr = nums.map(v => target - v);
  for (const i in subArr) {
    index = nums.indexOf(subArr[i]);
    if (index > 0 && index != i) {
      answer.push(Number(i), index);
      break;
    }
  }
  return answer;
}

console.log(sol([2, 7, 11, 15], 9));
